.github/workflows/build-mac.yml

```diff
@@ -0,0 +1,69 @@
+name: Mac tests
+
+on:
+  # This avoids having duplicate builds for a pull request
+  push:
+    branches:
+    - master
+    - 5.x
+    - 4.x
+  pull_request:
+    branches:
+    - master
+    - 5.x
+    - 4.x
+
+jobs:
+  cleanup-previous-runs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: rokroskar/workflow-run-cleanup-action@master
+      env:
+        GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+
+  build:
+    name: Py${{ matrix.PYTHON_VERSION }}, ${{ matrix.INSTALL_TYPE }}, ${{ matrix.TEST_TYPE }}
+    runs-on: macos-latest
+    env:
+      CI: 'true'
+      CODECOV_TOKEN: "56731c25-9b1f-4340-8b58-35739bfbc52d"
+      OS: 'macos'
+      PYTHON_VERSION: ${{ matrix.PYTHON_VERSION }}
+      RUN_SLOW: ${{ matrix.TEST_TYPE == 'slow' }}
+      USE_CONDA: ${{ matrix.INSTALL_TYPE == 'conda' }}
+    strategy:
+      fail-fast: false
+      matrix:
+        INSTALL_TYPE: ['conda']
+        PYTHON_VERSION: ['3.7', '2.7']
+        TEST_TYPE: ['fast', 'slow']
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v1.2.0
+      - name: Install Conda
+        uses: goanpeca/setup-miniconda@v1
+        with:
+           activate-environment: test
+           auto-update-conda: true
+           python-version: ${{ matrix.PYTHON_VERSION }} 
+      - name: Create test environment
+        shell: bash -l {0}
+        run: |
+          bash -l .github/workflows/install.sh
+      - name: Show test environment
+        shell: bash -l {0}
+        run: |
+          conda info
+          conda list
```

Comment:
Please run `check-manifest` here too. There are file types that we could per OS, so it's better to run it in all OSes.